src/styles/theme.ts

```diff
@@ -117,7 +117,7 @@ const theme = createTheme({
           props: { variant: 'head' },
           style: {
             textTransform: 'uppercase',
-            [breakpoints.between('xs', 'xl')]: { textAlign: 'left' },
+            [breakpoints.between('xs', 'xl')]: { textAlign: 'left', zIndex: 'auto' },
```

Comment:
I made this addition so the popper containing the study tags goes on top of the sticky table header on certain screen sizes.